src/helpers/infobox.js

```diff
@@ -19,7 +19,7 @@ export function insertInfoBox(text) {
   const div = document.createElement('div');
   div.style =
     'margin-left:20px; margin-bottom: 20px;background:#eee;border-radius:10px;padding:20px;box-shadow:#111 0px 5px 40px;cursor:pointer';
-  div.innerHTML = `<img src="${window.x8675309bp}icon/48.png" style="width:24px;height:24px;margin-right:8px"><span style="display:inline"></span>`;
+  div.innerHTML = `<img src="https://raw.githubusercontent.com/FastForwardTeam/FastForward/main/src/icon/48.png" style="width:24px;height:24px;margin-right:8px"><span style="display:inline"></span>`;
```

Comment:
use the internal one packed with the extension

Reply:
How do I do this? The extension ID function isn't here and the old one depended on {window.x8675309bp}, which we know doesn't work anymore.

Reply:
Can't file://${pwd} be used?